charts/gardener-extension-provider-aws/templates/vpa.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.vpa.enabled}}
+{{- if .Values.vpa.enabled }}
 apiVersion: "autoscaling.k8s.io/v1beta2"
```

Comment:
My initial intention was to update the vpa resources in the charts for the extension and admission controller in a separate pr where I also adjust their limits and request.

I did this now also here: d85bad998597c51ae2dd5f2f4d9fc60d32f7ba59

After this I only see old vpa resources in `./example/20-crd-vpa.yaml`.
Actually I do not recall where this is generated. It's not via `example/doc.go`. There we just generate the Gardener related CRDs

Reply:
> After this I only see old vpa resources in ./example/20-crd-vpa.yaml.

This is probably not generated and should be updated manually? Although it could also perhaps stay as it is, I don't think it's used unless you are creating a test environment manually, and there are probably better ways to do it than to use these CRDs here.